lib/puma/server.rb

```diff
@@ -595,6 +595,24 @@ def handle_request(req, lines)
       env[RACK_INPUT] = body
       env[RACK_URL_SCHEME] =  env[HTTPS_KEY] ? HTTPS : HTTP
 
+      if @early_hints
+        env[EARLY_HINTS] = lambda { |headers|
+          fast_write client, "HTTP/1.1 103 Early Hints\r\n".freeze
+
+          headers.each_pair do |k, vs|
+            if vs.respond_to?(:to_s) && !vs.to_s.empty?
+              vs.to_s.split(NEWLINE).each do |v|
+                fast_write client, "#{k}: #{v}\r\n"
+              end
+            else
+              fast_write client, "#{k}: #{v}\r\n"
```

Comment:
Should this be `fast_write client, "#{k}: #{vs}\r\n"` instead?
I don't see `v` defined in this level

Reply:
v is defined right here https://github.com/puma/puma/pull/1403/files/ea37ada76937ff6b3988e952f598c26cf373904e#diff-ef4c610f0ce605eb90f2810856394fd5R604

Reply:
Oh i see what you're saying - I will fix it